cmd/api/main.go

```diff
@@ -42,28 +43,33 @@ func main() {
-		GoogleOAuthClientID          string `conf:"google-oauth-client-id,noredact"`
-		MicrosoftOAuthClientID       string `conf:"microsoft-oauth-client-id,noredact"`
-		MicrosoftOAuthClientSecret   string `conf:"microsoft-oauth-client-secret"`
-		MicrosoftOAuthRedirectURI    string `conf:"microsoft-oauth-redirect-uri,noredact"`
-		ResendAPIKey                 string `conf:"resend-api-key"`
-		EmailChallengeFrom           string `conf:"email-challenge-from,noredact"`
-		EmailVerificationEndpoint    string `conf:"email-verification-endpoint,noredact"`
-		SegmentWriteKey              string `conf:"segment-write-key"`
-		FlyioAPIKey                  string `conf:"flyio-api-key"`
-		FlyioAuthProxyAppID          string `conf:"flyio-authproxy-app-id,noredact"`
-		FlyioAuthProxyAppCNAMEValue  string `conf:"flyio-authproxy-app-cname-value,noredact"`
-		FlyioAdminProxyAppID         string `conf:"flyio-adminproxy-app-id,noredact"`
-		FlyioAdminProxyAppCNAMEValue string `conf:"flyio-adminproxy-app-cname-value,noredact"`
-		AdminLogosS3BucketName       string `conf:"admin-logos-s3-bucket-name,noredact"`
+		SentryDSN                             string `conf:"sentry-dsn,noredact"`
+		SentryEnvironment                     string `conf:"sentry-environment,noredact"`
+		ServeAddr                             string `conf:"serve-addr,noredact"`
+		DB                                    string `conf:"db"`
+		DefaultAuthURL                        string `conf:"default-auth-url,noredact"`
+		DefaultAdminSetupURL                  string `conf:"default-admin-setup-url,noredact"`
+		PageEncodingValue                     string `conf:"page-encoding-value"`
+		SAMLStateSigningKey                   string `conf:"saml-state-signing-key"`
+		GoogleOAuthClientID                   string `conf:"google-oauth-client-id,noredact"`
+		MicrosoftOAuthClientID                string `conf:"microsoft-oauth-client-id,noredact"`
+		MicrosoftOAuthClientSecret            string `conf:"microsoft-oauth-client-secret"`
+		MicrosoftOAuthRedirectURI             string `conf:"microsoft-oauth-redirect-uri,noredact"`
+		ResendAPIKey                          string `conf:"resend-api-key"`
+		EmailChallengeFrom                    string `conf:"email-challenge-from,noredact"`
+		EmailVerificationEndpoint             string `conf:"email-verification-endpoint,noredact"`
+		SegmentWriteKey                       string `conf:"segment-write-key"`
+		CloudflareAPIKey                      string `conf:"cloudflare-api-key"`
+		CustomAuthDomainCloudflareZoneID      string `conf:"custom-auth-domain-cloudflare-zone-id,noredact"`
+		CustomAuthDomainCloudflareCNameValue  string `conf:"custom-auth-domain-cloudflare-cname-value,noredact"`
+		CustomAdminDomainCloudflareZoneID     string `conf:"custom-admin-domain-cloudflare-zone-id,noredact"`
+		CustomAdminDomainCloudflareCNameValue string `conf:"custom-admin-domain-cloudflare-cname-value,noredact"`
+		FlyioAPIKey                           string `conf:"flyio-api-key"`
```

Comment:
Do we still need these for any reason?

Reply:
Thinking of deleting fly.io stuff only after this is rolled out.